### PR TITLE
Enable multiple-day allDay events (Android)

### PR DESCRIPTION
**Pull request by FCView**
Fix for issue #220 

- Enables mutiple-day allDay events on Android. 
- Updates the Example to add the ability to set the end date for an allDay event.
- Updates the EventItem class in the Example to show how to display the date for an allDay event since the Android Calendar Provider returns the time (UTC midnight) adjusted to the local timezone, which can cause an error displaying the correct date.

The changes in the Example should only affect Android, but I am not developing on iOS, so I am unable to test on iOS to make sure these changes in the didn't inadvertently affect anything.

### CODE REVIEW

**Comment by FCView** [comment]
I'm still new to this and am not familiar with those checks, so I'm not sure why they are showing not successful. It looks like maybe Flutter needs to be updated and there was an error message about the version not being an integer. Other than that, if you need anything more from me, just let me know.

**Comment by nickrandolph** [comment]
@FCView  apologies for the delay - I'll be trying to review this PR this week

File: example/lib/presentation/event_item.dart [hunk 1]
```diff
@@ -25,6 +27,7 @@ class EventItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    print(_calendarEvent.title);
```
Comment (matejdro) [inline example/lib/presentation/event_item.dart:30]: Should this be removed before it's merged in?

**Comment by matejdro** [comment]
I've tried this out and it seems to work fine.

**Comment by hyouuu** [comment]
@nickrandolph are we going to merge this in? Still having issues with all day events even if it's a single day, as mentioned before in https://github.com/builttoroam/device_calendar/issues/220 and https://github.com/builttoroam/device_calendar/issues/164

**Comment by thomassth** [comment]
@hyouuu we are in the process of migrating to null-safety, can you edit your commit to make it null safe as well?
Also, please consider pulling the dev branch into your commit, which will help your merge request pass the auto checks.

**Comment by hellcp** [comment]
I rebased this and updated for null safety in my own branch, feel free to use that for this PR https://github.com/hellcp/device_calendar/tree/multiday

**Comment by hyouuu** [comment]
@nickrandolph could you help check @hellcp 's branch and other stuff? Thanks!

**Comment by NoahAndrews** [comment]
Why was this closed?

**Comment by FCView** [comment]
Replaced with PR #324

**Pull request closed without merging**